tests/testapps/CTCatalog/CTCatalog/Tests/CTCFrameTestViewController.mm

```diff
@@ -190,26 +209,29 @@ - (void)refreshValuesForFrame:(CTFrameRef)frame {
     for (CFIndex i = 0; i < count; ++i) {
         cell = [[UITableViewCell alloc] initWithStyle:UITableViewCellStyleDefault reuseIdentifier:@"FrameLines"];
         cell.textLabel.text = [NSString stringWithFormat:@"X: %f, Y: %f", lineOrigins[i].x, lineOrigins[i].y];
-        CTFrameLineTestView* textLabel = [[CTFrameLineTestView alloc] initWithFrame:CGRectMake(0, 0, 400, 60)];
+        CTFrameLineTestView* textLabel = [[CTFrameLineTestView alloc] initWithFrame:CGRectMake(0, 0, 2 * width / 3, 60)];
         textLabel.backgroundColor = [UIColor whiteColor];
         textLabel.lineRef = static_cast<CTLineRef>(CFArrayGetValueAtIndex(lines, i));
+        CFRetain(textLabel.lineRef);
```

Comment:
If it is intended to be retained, it should be retained by the `lineRef` setter. This applies equally to every instance of `x.y = z; Retain(x.y);`. The burden belongs on the callee, not the caller.